### ISSUE TITLE
There are 4 extra (null) pane in Inspect after right clicking on the form icon

### ISSUE THREAD

**Issue by Vino-Wang** [dotnet/winforms#3699]
* .NET Core Version: 
5.0.100-rc.1.1.20405.11

* Have you experienced this same bug with .NET Framework?: 
Yes

**More info:**
This is not a regression issue.

**Problem description:**
There are 4 extra (null) pane in Inspect after right clicking on the form icon.
![ExtraPane](https://user-images.githubusercontent.com/18568603/89481805-1e9a8f00-d7cb-11ea-8eff-06eac760a00f.png)
![ExtraItem](https://user-images.githubusercontent.com/18568603/89481824-2c501480-d7cb-11ea-9e5f-51c2bd45908b.gif)

**Expected behavior:**
There should have no extra (null) panel.

**Minimal repro:**
1.	Create a winforms core project.
2.	Build and run.
3.	Launch the Inspect tool and right click on the form icon, observe the left items in Inspect.
Or 
1.	Run the attached project. [CustomAccessibleNameForForm.zip](https://github.com/dotnet/winforms/files/5032104/CustomAccessibleNameForForm.zip)
2.	Build and run.
3.	Launch the Inspect tool and right click on the form icon, observe the left items in Inspect.

**Comment by JeremyKuhne** [comment]
This is a normal Windows behavior- got this to repro with a basic Windows app that doesn't use WinForms.
